sktime/classification/distance_based/_proximity_forest.py

```diff
@@ -1539,17 +1805,43 @@ def get_test_params(cls, parameter_set="default"):
             instance.
             ``create_test_instance`` uses the first (or only) dictionary in ``params``.
         """
+        from sktime.utils.validation._dependencies import _check_soft_dependencies
+
         if parameter_set == "results_comparison":
             return {"n_estimators": 3, "max_depth": 2, "n_stump_evaluations": 2}
         else:
-            param1 = {"n_estimators": 2, "max_depth": 1, "n_stump_evaluations": 1}
```

Comment:
can you please restore these? parameter sets should never be empty.

You can of course remove the "else" indentation, since if ends in a return.

Reply:
these are the general tests run in the future to check whether the estimator runs, so if you want to check only locally, use `check_estimator` or `pytest -k` to subset but do not change the test cases.